docs/content/guides/dagster-pipes/aws-lambda.mdx

```diff
@@ -0,0 +1,288 @@
+        # multi-assets you can pass an `asset_key` parameter.
+
+        pipes.report_asset_materialization(
+            metadata={
+                "some_metric": {"raw_value": some_parameter_value + 1, "type": "int"}
+            },
+            data_version="alpha",
+        )
+```
+
+Let's review what this code does:
+
+- Imports `PipesMappingParamsLoader` and <PyObject object="open_dagster_pipes" module="dagster_pipes" /> from `dagster_pipes`
+
+- **Initializes the Dagster Pipes context (<PyObject object="open_dagster_pipes" module="dagster_pipes" />), which yields an instance of <PyObject object="PipesContext" module="dagster_pipes" /> called `pipes`.**
+
+  <!-- TODO: Looks like this paragraph will need to be updated a bit, based on the fact that we're using PipesMappingParamsLoader -->
+
+  We're using the default context loader (<PyObject object="PipesDefaultContextLoader" module="dagster_pipes" />) and message writer (<PyObject object="PipesDefaultMessageWriter" module="dagster_pipes" />) in this example. These objects establish communication between the orchestration and external process. On the orchestration end, these match a corresponding `PipesLambdaEventContextInjector` and `PipesLambdaLogsMessageReader`, which are instantiated inside the <PyObject module="dagster_aws" object="PipesLambdaClient" />.
+
+- **Inside the body of the context manager (<PyObject object="open_dagster_pipes" module="dagster_pipes" />), retrieve a log and report an asset materialization.** These calls use the temporary communications channels established by <PyObject object="PipesDefaultContextLoader" module="dagster_pipes" /> and <PyObject object="PipesDefaultMessageWriter" module="dagster_pipes" />. To see the full range of what you can do with the <PyObject object="PipesContext" module="dagster_pipes" />, see the API docs or the general [Pipes documentation](/guides/dagster-pipes).
+
+At this point you can execute the rest of your AWS Lambda code as normal, invoking various <PyObject object="PipesContext" module="dagster_pipes" /> APIs as needed.
+
+### Step 1.3: Deploy the function
+
+When finished, click the **Deploy** button to update and deploy the function.
+
+---
+
+## Step 2: Create the Dagster objects
+
+In this step, you’ll create a Dagster asset that, when materialized, opens a Dagster pipes session and invokes the Lambda function you created in [Step 1](#step-1-create-a-lambda-function).
+
+### Step 2.1: Define the Dagster asset
+
+In your Dagster project, create a file named `dagster_lambda_pipes.py` and paste in the following code:
+
+```python
+# dagster_lambda_pipes.py
+
+from dagster import AssetExecutionContext, Definitions, asset
+from dagster_aws import PipesLambdaClient
+
+
+@asset
+def lambda_pipes_asset(context: AssetExecutionContext, lambda_pipes_client: PipesLambdaClient):
+  return lambda_pipes_client.run(
+      context: context,
+      client: "TODO",
```

Comment:
Do we have an example of a value that would go here?

Reply:
`boto3.client("lambda")` with a `import boto3` up in the imports section